stages/5-cluster-connect-check/create-apikey-check

```diff
@@ -41,4 +41,11 @@ then
     exit 1; 
 else
     bash utils/e2e-cr jobname:create-apikey-check jobphase:Completed
+    # saving secret yaml into a file
+    kubectl get secret director-user-pass -n litmus -oyaml >> secret.yaml
```

Comment:
@harshita-sharma011 `>` should be used instead of `>>` here. Merging it for now.